core/bor_events.go

```diff
@@ -8,3 +8,10 @@ import (
 type StateSyncEvent struct {
 	Data *types.StateSyncData
 }
+
```

Comment:
Add add here a specific type for each of the new chain events:
```
var (
    ChainHeadReorgEvent = "reorg"
    ChainHeadCanonicalEvent = "head"
    ChainHeadForkEvent = "fork"
)
```
Use these names instead of using a raw string each time you want to reference each event.